src/javascripts/components/City.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+
+class City extends Component {
```

Comment:
@forsaken1 

- make component refactoring (City/Team/User). Look like - it can be single component **EditableLabel**
- add PropTypes
- don't use data structures - which will be not used (data...). Component has to accept only required data, nothing more